### PR TITLE
Add crucible roles to README

### PR DESCRIPTION
**Pull request by nocturnalastro**
Test-Hints: no-check

### CODE REVIEW

File: roles/mount_discovery_iso_for_pxe/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# mount_discovery_iso_for_pxe
\ No newline at end of file
+# mount_discovery_iso_for_pxe
+
+Extracts the required artifects for a pxe boot from a discovery iso
```
Comment (tonyskapunk) [inline roles/mount_discovery_iso_for_pxe/README.md:3]:
```suggestion
Extracts the required artifacts for a pxe boot from a discovery iso
```

**Review by tonyskapunk — COMMENTED** [review]
All good, just a typo

**Review by thekad — APPROVED** [review]
:shipit:

**Comment by fredericlepied** [comment]
Could yoy rebase your change to have the correct CI jobs? DCI should not be started on a doc change.

**Comment by nocturnalastro** [comment]
I've rebased lets see if the trigger works correctly :)

**Comment by fredericlepied** [comment]
> I've rebased lets see if the trigger works correctly :)
> 
> Edit: looks like its still triggered as the DCI job is pending.
> 
> Edit2: I checked the tree looks like this check isn't working correctly
> 
> https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/58581e425d856ffbbd472dde7bd77337bec96ac5/hack/maybe-dci-check-change#L40
> 
> Here is the workflow run which shows it going into pending.
> https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/7460111555/job/20297517979?pr=117

There is a code file in the change (roles/process_kvm_nodes/tasks/create_node.yml) so it is behaving correctly...

**Comment by nocturnalastro** [comment]
> > I've rebased lets see if the trigger works correctly :)
> > Edit: looks like its still triggered as the DCI job is pending.
> > Edit2: I checked the tree looks like this check isn't working correctly
> > https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/58581e425d856ffbbd472dde7bd77337bec96ac5/hack/maybe-dci-check-change#L40
> > 
> > Here is the workflow run which shows it going into pending.
> > https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/7460111555/job/20297517979?pr=117
> 
> There is a code file in the change (roles/process_kvm_nodes/tasks/create_node.yml) so it is behaving correctly...

Oh so there is. :facepalm:

**Pull request merged**